scripts/s3-sanity/s3-sanity-test.sh

```diff
@@ -17,81 +18,301 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-#!/bin/bash
-
-set -e
 
 #########################
 # S3 sanity test script #
 #########################
 
-USAGE="USAGE: bash $(basename "$0") [--help]
+ldappasswd=''
```

Comment:
Restructure the logic for s3iamclitestson s3cmdtestson

Reply:
Please review the current code which is slightly restructured.